staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -429,7 +429,7 @@ func setObjectSelfLink(ctx context.Context, obj runtime.Object, req *http.Reques
 		return err
 	}
 	if err := namer.SetSelfLink(obj, uri); err != nil {
-		klog.V(4).Infof("Unable to set self link on object: %v", err)
```

Comment:
I don't know how often this happens so maybe there was a reason to make it `V(4)`. There's probably now `V().ErrorS`, so either logging an error here everytime is fine or it should be an InfoS.
Seems to be a warning in this case so maybe: `klog.V(4).InfoS("Unable to set self link on object", "error", err)`.
Could also add the `klog.KObj(obj)` newly introduced here.

Reply:
I have updated the code to klog.V(4).InfoS(), however, klog.KObj(obj) does no work here since I can't get the name and namespace of obj.